share/picongpu/examples/WeibelTransverse/include/picongpu/simulation_defines/param/particle.param

```diff
@@ -76,29 +74,29 @@ namespace manipulators
          */
         static constexpr float_64 temperature = 0.005;
     };
-    /* definition of SetDrift start */
+    /* definition of SetDrift start
     using AddTemperature = unary::Temperature<TemperatureParam>;
 
-} //namespace manipulators
+} // namespace manipulators
 
 
 namespace startPosition
 {
 
-    struct QuietParam
+    struct QuietParam4ppc
     {
         /** Count of particles per cell per direction at initial state
          *  unit: none */
```

Comment:
since you're at it, the closing token `*/` should be on a new line, right?

Reply:
these lines were all intentionally not  touched in this PR.
I can commit it now, but this will obfuscate the rebase in this PR with unnecessarily changed lines. as you wish...